src/kata3.numberToReversedDigits.js

```diff
@@ -1,3 +1,8 @@
-const numberToReversedDigits = (number) => {};
+function numberToReversedDigits(number) {
+  const newArray = Array.from(String(number));
+  const reversedArray = newArray.reverse();
+  const outputArray = reversedArray.map((item) => Number(item));
+  return outputArray;
```

Comment:
The logic here is sound, but all these method calls can be chained together without all the variable creation:
```
  return Array.from(String(number))
    .reverse()
    .map((item) => Number(item));
```